lib/workers/repository/config-migration/branch/migrated-data.ts

```diff
@@ -61,6 +100,7 @@ export class MigratedDataFactory {
         content = JSON.stringify(migratedConfig, undefined, indent);
       }
 
+      content = await applyPrettierFormatting(content);
```

Comment:
i think we need to pass the `ident` variable to prettier, otherwise it will use the default values, which is probably wrong.